src/app/components/hris/employees/employee-profile/accordions/accordion-banking/accordion-banking.component.ts

```diff
@@ -78,9 +78,7 @@ export class AccordionBankingComponent {
           console.warn("No banking details available.");
```

Comment:
Not sure about the console.warn and if it should be a error ? 👍🏻

Reply:
So that is not the focus on what was changed. We have many console stuff that needs to be removed.

Reply:
Have gone and fixed it :)